recipe/meta.yaml

```diff
@@ -75,15 +76,15 @@ outputs:
         - psy-simple
         - pyproj >=2.1
         - pys2index  # only from conda-forge
-        - python >=3.9,<3.12
+        - python >={{ python_min }},<3.12
```

Comment:
thanks for looking at this @bouweandela - this min thing is so clunky, at least they should have allowed a max variable - maybe we should do it ourselves? Or not :grin:

Reply:
Like this? 6cba2b6

Reply:
yes that looks great actually! Nice and centralized :+1: